packages/uniforms-mui/src/DateField.tsx

```diff
@@ -35,8 +39,11 @@ function Date({
   readOnly,
   showInlineError,
   value,
+  type,
   ...props
 }: DateFieldProps) {
+  const dateType = type === 'date' ? type : 'datetime-local';
```

Comment:
This was included in the initial implementation so I added it here for consistency but I'm not sure we need this logic. Thoughts @radekmie ?